package.json

```diff
@@ -97,12 +97,17 @@
   "scripts": {
     "start": "craco start",
     "build": "craco build",
-    "test": "craco test --env=jsdom",
-    "test:debug": "craco --inspect-brk test --runInBand --no-cache",
+    "test": "NODE_PATH=src/custom craco test --env=jsdom",
+    "test:debug": "NODE_PATH=src/custom craco --inspect-brk test --runInBand --no-cache",
     "eject": "react-scripts eject",
     "integration-test": "start-server-and-test 'serve build -l 3000' http://localhost:3000 'cypress run'",
     "dedup": "npx yarn-deduplicate"
   },
+  "jest": {
+    "moduleNameMapper": {
+      "@src/(.*)": "<rootDir>/src/$1"
+    }
+  },
```

Comment:
Path resolution when running tests stopped working when I added `custom/xdai`. Likely because it didn't have a mirror in `src/`.